src/claimDetail.js

```diff
@@ -59,83 +85,811 @@ function extractData() {
+            break;
+          case "GEN_M01":
+            title = "Missing medical information";
+            body = html`<div>
+              We need a medical certificate from your doctor (or
+              <a
+                href="https://www.nj.gov/labor/myleavebenefits/worker/resources/approvedmedicalpractitioners.shtml"
+                target="_blank"
+                >other approved medical provider</a
+              >) to review your claim.<br /><br />
+              <b>Steps to complete</b>
+              <ul
+                style="padding-inline-start: 20px; margin-block-start: 0; margin-block-end: 0"
+              >
+                <li>
+                  Share the medical certificate instructions with your doctor.
+                  The document is listed under "Medical Certificate
+                  Instructions" in
+                  <a href="${claimDocsUrl}" target="_blank">claim documents</a>.
+                </li>
+                <li>
+                  Make sure your doctor completes this step by
+                  <b>${onlineResponseDateFormatted}</b> so your claim isn't
+                  delayed or denied. To confirm we received it, check under
+                  "Medical Summary" in your
+                  <a href="${claimDocsUrl}" target="_blank">claim documents</a>.
+                </li>
+              </ul>
+            </div>`;
+            break;
+          case "GEN_W01":
+            title = "Missing workers' compensation information";
+            body = html`<div>
+              We need a medical certificate from your doctor (or
+              <a
+                href="https://www.nj.gov/labor/myleavebenefits/worker/resources/approvedmedicalpractitioners.shtml"
+                target="_blank"
+                >other approved medical provider</a
+              >) to review your claim.<br /><br />
+              <b>Steps to complete</b>
+              <ul
+                style="padding-inline-start: 20px; margin-block-start: 0; margin-block-end: 0"
+              >
+                <li>
+                  Download and print the document under "Claimant Information
+                  Forms" in your
+                  <a href="${claimDocsUrl}" target="_blank">claim documents</a>.
+                  This document lists all the information we need.
+                </li>
+                <li>
+                  Make sure you complete the form, and fax or mail it to us by
+                  <b>${onlineResponseDateFormatted}</b> so your claim isn't
+                  delayed or denied.
+                </li>
+              </ul>
+            </div>`;
+            break;
+          case "SENT_C10":
+            title = "Missing claimant information";
+            body = html`<div>
+              We need claimant information from you to review your claim. On
+              ${mailDateFormatted}, we mailed a Request to Claimant for
+              Information (C10) to your address on file: ${note.sentTo}.<br /><br />
```

Comment:
Dust: just nothing that since `note.sentTo` resolves to the person's name, it may be a bit confusing to read something like `On August 25, 2023, we mailed a Request to Claimant for Information (C10) to your address on file: Jenni Mahlstedt.` (this exists in other parts of this function as well). Given that the page doesn't seem to render an address I assume this is by design but wanted to call it out

Reply:
Oh shoot thanks for catching this! I thought we had updated, but perhaps not all the scenarios